src/components/services-view/service-view.classes.ts

```diff
@@ -0,0 +1,181 @@
+import {allCategories} from '../../constants';
+import * as _ from 'lodash';
+
+interface IServiceItem {
+    icon: string;
+    name: string;
+    category: string;
+    subCategory: string;
+    description: string;
+    longDescription: string;
+    data: any;
+}
+
+export class CatalogServiceItem implements IServiceItem {
+    public icon: string;
+    public name: string;
+    public category: string;
+    public subCategory: string;
+    public description: string;
+    public longDescription: string;
+    public data: any;
+
+    constructor (catalogService: any) {
+        this.data = catalogService;
+        this.icon = this.getIcon();
+        this.name = this.getName();
+        this.description = this.getDescription();
+        this.longDescription = this.getLongDescription();
+        this.subCategory = this.getSubCategory();
+        this.category = this.getCategory();
```

Comment:
Hi, fixed ServiceClasses to be under multiple categories and sub-categories.  Waiting on Jeff's PR in order to use relevant filters for the builder images.